backend/controllers/event.controller.js

```diff
@@ -0,0 +1,66 @@
+const { Event } = require('../models');
+
+const EventController = {};
+
+EventController.event_list = async function (req, res) {
+  const { query } = req;
+
+  try {
+    const events = await Event.find(query);
+    return res.status(200).send(events);
+  } catch (err) {
+    return res.sendStatus(400);
```

Comment:
I think enums should be used for the return codes. It will provide readable text for each code, which would help with preventing coding errors. This package contains all the codes, maybe use this...  https://www.npmjs.com/package/http-status-codes

Reply:
👍